code/__HELPERS/time.dm

```diff
@@ -3,9 +3,11 @@
 #define MINUTES * 600
 #define HOURS   * 36000
 
-//Returns the world time in english
-proc/worldtime2text(timestamp = world.time)
-	return "[round(timestamp / 36000) + 12]:[(timestamp / 600 % 60) < 10 ? add_zero(timestamp / 600 % 60, 1) : timestamp / 600 % 60]"
+/proc/timestamp2text(timestamp = 0, hour_to_add = 0)

```

Comment:
Sorry, let me try again:

in order to preserve current functionality, shouldn't hour_to_add default to 12 because that's what the current code does?